air/src/proof/mod.rs

```diff
@@ -32,6 +32,7 @@ pub use table::Table;
 // ================================================================================================
 
 const GRINDING_CONTRIBUTION_FLOOR: u32 = 80;
+const PROXIMITY_PARAMETER_UPPER: u64 = 1000;
```

Comment:
A couple of minor nits:

- I'd probably name this `MAX_PROXIMITY_PARAMETER`.
- Let's add `#[cfg(feature = "std")]` attribute for this constant.

Reply:
Done!